src/rs_distributions/transforms/fill_scale_tril.py

```diff
@@ -0,0 +1,84 @@
+import torch
+from torch.distributions import Transform, constraints
+from torch.distributions.transforms import SoftplusTransform
+from torch.distributions.utils import vec_to_tril_matrix, tril_matrix_to_vec
+
+
+class FillScaleTriL(Transform):
```

Comment:
I request you add a `params_size` staticmethod a la tfp's multivariate normal: https://www.tensorflow.org/probability/api_docs/python/tfp/layers/MultivariateNormalTriL#params_size

Reply:
Something like 

```python
@staticmethod
def params_size(event_size):
# for lower triangular we need n*(n+1)//2 elements
return event_size * (event_size + 1) // 2 
```

Reply:
yeah that's what i am thinking. it's helpful to have when you are in dimensions greater than 3.